Grenache.Example.Client/Program.cs

```diff
@@ -4,38 +4,59 @@
 
 namespace Grenache.Example.Client

```

Comment:
I will come up with a strange request, can we keep indentation to 2 spaces please?

Reply:
for c# strange indeed. addressed